src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -417,16 +417,18 @@ protected void Stop()
                     return;
                 }
 
-                await _editorSession.PowerShellContext.ExecuteScriptStringAsync(
+                if (attachParams.ProcessId != "current") {
```

Comment:
Shouldn't need this if because the outer `if` tries to parse the ProcessId as an integer (L411):

```csharp
if (processIdIsSet && int.TryParse(attachParams.ProcessId, out int processId) && (processId > 0))
```

Reply:
Good catch. Updated the PR.